src/elections/urls.py

```diff
@@ -0,0 +1,160 @@
+import base64
+import logging
+import os
+import re
+import urllib.parse
+from datetime import datetime
+from enum import Enum
+
+import auth
+import auth.crud
+import database
+import elections
+import requests  # TODO: make this async
+import xmltodict
+from constants import root_ip_address
+from fastapi import APIRouter, BackgroundTasks, FastAPI, HTTPException, Request, status
+from fastapi.exceptions import RequestValidationError
+from permission import types
+
+_logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix="/elections",
+    tags=["elections"],
+)
+
+class ElectionType(Enum):
+    GENERAL_ELECTION = "general_election"
+    BY_ELECTION = "by_election"
+    COUNCIL_REP_ELECTION = "council_rep_election"
```

Comment:
I have a personal grudge against python enums which may or may not be reasonably motivated; they often only get converted into string, and code gets polluted with `x.value`. I bring this up especially since if their purpose is to be stored in the database, that involves converting them into string (in the backend).

It would be better to just have them be string constants in the first place. This approach has a few small negatives, but ultimately reduces complexity, as you never need to worry about whether an instance contains a string of the enum, or the enum itself.